docs/howtos/quick-start/cli/_cli-basic-querying.mdx

```diff
@@ -0,0 +1,100 @@
+
+- Connect to Redis using CLI or **[RedisInsight](https://redis.io/docs/stack/insight/)** (a GUI tool to visualize data & run commands)
+
+**RedisInsight**
+![RedisInsight](../images/redis-insight.svg)
+
+```shell
+# syntax 1 : connect using host & port
+$ redis-cli -h host -p port -a password
+
+# example 1
+$ redis-cli -h redis15.localnet.org -p 6390 -a myUnguessablePassword
```

Comment:
entering the password with the -a command gives a warning, might not be the best experience getting started. Maybe show the AUTH command directly after connecting w/ host and port to avoid this?

```
$ redis-cli -h redis15.localnet.org -p 6390
> AUTH myUnguessablePassword
```